### PR TITLE
mgit: Init at 0.1.10

### PR DESCRIPTION
**Pull request by koozz**
###### Motivation for this change

Commandline tool written in Rust by myself.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/misc/mgit/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    sha256 = "0lsph259vwxrj8nprkz6dr2nahbl7h037zi6z8w8zgpcg0fs17y6";
+  };
+
+  cargoSha256 = "1a0sznzn1vmavs4l3wmpx3qjh98sknz3fjybsmicch25z3j75iyi";
+
+  meta = with stdenv.lib; {
+    description = "A utility that performs git actions on multiple directories within the current tree";
+    homepage = "https://github.com/koozz/mgit";
+    license = licenses.mit;
+    maintainers = with maintainers; [ koozz ];
```
Comment (sikmir) [inline pkgs/development/tools/misc/mgit/default.nix:20]: You should add yourself to maintainder-list.nix.

Comment (koozz) [reply pkgs/development/tools/misc/mgit/default.nix:20]: I'm sorry, I missed that. Just added a commit.

**Review by sikmir — APPROVED** [review]

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -9203,4 +9203,10 @@
     github = "deifactor";
     githubId = 30192992;
   };
+  koozz = {
```
Comment (timstott) [inline maintainers/maintainer-list.nix:9206]: :abcd: Users are mostly inserted by alphabetical order. Could you find the next best spot for your username.

**Review by timstott — CHANGES_REQUESTED** [review]
:raising_hand: You need to add your package to `pkgs/top-level/all-packages.nix`

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (koozz) [reply maintainers/maintainer-list.nix]: I see, the last few were totally unordered and that made me think it was chronological. I've put myself in the ordered part now.

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -4366,6 +4366,12 @@
     githubId = 15692230;
     name = "Muhammad Herdiansyah";
   };
+  koozz = {
+    name = "Jan van den Berg";
+    email = "264371+koozz@users.noreply.github.com";
+    github = "koozz";
+    githubId = 264371;
```
Comment (timstott) [inline maintainers/maintainer-list.nix:4373]: I believe the `githubId` is off by one :grimacing: 
```
curl -s https://api.github.com/users/koozz | jq .id
264372
```

Comment (koozz) [reply maintainers/maintainer-list.nix]: Weird, I found the noreply-email in my GitHub settings and copied that. Fixed it now.

**Review by timstott — APPROVED** [review]

**Pull request closed without merging**